catalog/rocks/validate.go

```diff
@@ -191,5 +191,16 @@ func ValidateRequiredString(v interface{}) error {
 	return nil
 }
 
+func ValidatePositiveInt(v interface{}) error {
+	i, ok := v.(int)
+	if !ok {
+		panic(ErrInvalidType)
+	}
+	if i <= 0 {
```

Comment:
< 0? Thought this parameter can be 0 in case the client didn't pass any decision in the request.

Reply:
Oops! fixed